rules/common/src/main/kotlin/io/nlopez/compose/rules/ParameterNaming.kt

```diff
@@ -17,16 +17,20 @@ class ParameterNaming : ComposeKtVisitor {
         // E.g. onClick, onTextChange, onValueChange, and a myriad of other examples in the compose foundation code.
 
         val lambdaTypes = function.containingKtFile.lambdaTypes(config)
+        val allowed = config.getSet("allowedLambda", emptySet())
```

Comment:
The name should be more specific, as these config names are supposed to cover all of the rules (because of ktlint). 

Something like allowedLambdaParameterNames would work.